### ISSUE TITLE
Boost UI message pool is broken in v1.6.0

### ISSUE THREAD

**Issue by LexLuthr** [filecoin-project/boost#1285]
Show all messages is broken with nil pointer error
<img width="1183" alt="Screenshot 2023-03-16 at 4 03 05 PM" src="https://user-images.githubusercontent.com/88259624/225590544-641fa373-929d-400a-a2c2-7b76f92c34d8.png">

**Comment by LexLuthr** [comment]
Logs:

```
2023/03/16 12:46:19 graphql: panic occurred: runtime error: invalid memory address or nil pointer dereference
goroutine 23679208 [running]:
github.com/graph-gophers/graphql-go/log.(*DefaultLogger).LogPanic(0x7c860e8?, {0x59cc0e8?, 0xc017e69890}, {0x4423400, 0x78aa780})
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/log/log.go:21 +0x65
github.com/graph-gophers/graphql-go/internal/exec.execFieldSelection.func2.1()
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/internal/exec/exec.go:187 +0x89
panic({0x4423400, 0x78aa780})
	/usr/local/go/src/runtime/panic.go:884 +0x212
github.com/filecoin-project/boost/gql.(*resolver).Mpool(0xc013650620, {0x59cc0e8, 0xc017e69980}, {0x19?})
	/root/boost/gql/resolver_mpool.go:70 +0x46d
reflect.Value.call({0x4812480?, 0xc013650620?, 0x30?}, {0x4866a5a, 0x4}, {0xc017e699b0, 0x2, 0x7d842e?})
	/usr/local/go/src/reflect/value.go:584 +0x8c5
reflect.Value.Call({0x4812480?, 0xc013650620?, 0x42b3e20?}, {0xc017e699b0?, 0x1e?, 0x0?})
	/usr/local/go/src/reflect/value.go:368 +0xbc
github.com/graph-gophers/graphql-go/internal/exec.execFieldSelection.func2(0x203000?, {0x59cc0e8?, 0xc017e69890?}, 0xc01abf7b18, 0xc01c9bcf40, 0xc01f077ec0, {0x59cc0e8?, 0xc017e69980})
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/internal/exec/exec.go:211 +0x4cb
github.com/graph-gophers/graphql-go/internal/exec.execFieldSelection({0x59cc0e8, 0xc017e69890}, 0xc01a7f5480, 0x0?, 0xc01c9bcf40, 0x0?, 0x1)
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/internal/exec/exec.go:231 +0x1ce
github.com/graph-gophers/graphql-go/internal/exec.(*Request).execSelections.func1(0xc01c9bcf40)
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/internal/exec/exec.go:81 +0x1b8
created by github.com/graph-gophers/graphql-go/internal/exec.(*Request).execSelections
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/internal/exec/exec.go:77 +0x1d0

context: context.Background.WithValue(type *http.contextKey, val <not Stringer>).WithValue(type *http.contextKey, val 127.0.0.1:8080).WithCancel.WithCancel.WithValue(type opentracing.contextKey, val <not Stringer>)
2023/03/16 12:46:22 graphql: panic occurred: runtime error: invalid memory address or nil pointer dereference
goroutine 23679287 [running]:
github.com/graph-gophers/graphql-go/log.(*DefaultLogger).LogPanic(0x7c860e8?, {0x59cc0e8?, 0xc0295c21e0}, {0x4423400, 0x78aa780})
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/log/log.go:21 +0x65
github.com/graph-gophers/graphql-go/internal/exec.execFieldSelection.func2.1()
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/internal/exec/exec.go:187 +0x89
panic({0x4423400, 0x78aa780})
	/usr/local/go/src/runtime/panic.go:884 +0x212
github.com/filecoin-project/boost/gql.(*resolver).Mpool(0xc013650620, {0x59cc0e8, 0xc0295c2300}, {0x19?})
	/root/boost/gql/resolver_mpool.go:70 +0x46d
reflect.Value.call({0x4812480?, 0xc013650620?, 0x30?}, {0x4866a5a, 0x4}, {0xc0295c2330, 0x2, 0x7d842e?})
	/usr/local/go/src/reflect/value.go:584 +0x8c5
reflect.Value.Call({0x4812480?, 0xc013650620?, 0x42b3e20?}, {0xc0295c2330?, 0x1e?, 0x0?})
	/usr/local/go/src/reflect/value.go:368 +0xbc
github.com/graph-gophers/graphql-go/internal/exec.execFieldSelection.func2(0x203000?, {0x59cc0e8?, 0xc0295c21e0?}, 0xc0224050b0, 0xc018fee000, 0xc00c729ec0, {0x59cc0e8?, 0xc0295c2300})
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/internal/exec/exec.go:211 +0x4cb
github.com/graph-gophers/graphql-go/internal/exec.execFieldSelection({0x59cc0e8, 0xc0295c21e0}, 0xc01156ca80, 0xc0212a5d80?, 0xc018fee000, 0x53b91942c179?, 0x1)
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/internal/exec/exec.go:231 +0x1ce
github.com/graph-gophers/graphql-go/internal/exec.(*Request).execSelections.func1(0xc018fee000)
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/internal/exec/exec.go:81 +0x1b8
created by github.com/graph-gophers/graphql-go/internal/exec.(*Request).execSelections
	/root/code/pkg/mod/github.com/graph-gophers/graphql-go@v1.3.0/internal/exec/exec.go:77 +0x1d0

context: context.Background.WithValue(type *http.contextKey, val <not Stringer>).WithValue(type *http.contextKey, val 127.0.0.1:8080).WithCancel.WithCancel.WithValue(type opentracing.contextKey, val <not Stringer>)
```

**Comment by LexLuthr** [comment]
It would seem the method was moved from package filcns to consensus in https://github.com/filecoin-project/lotus/commit/627d6e70a67fe12383eade11e879c259295768ad
Once we update the Lotus dependency to v1.20.3, we should be able to fix this as well.

**Comment by nyzell601** [comment]
i have same problem... do you fix it?

**Comment by jacobheun** [comment]
This should be fixed in the v1.6.1-rc1 via https://github.com/filecoin-project/boost/pull/1293.
